ts/components/RemindEmailValidationOverlay.tsx

```diff
@@ -5,39 +5,21 @@
 import I18n from "i18n-js";
 import { Button, Content, H2, Text, View } from "native-base";
 import * as React from "react";
-import { BackHandler, Image } from "react-native";
+import { Image } from "react-native";
 import TopScreenComponent from "./screens/TopScreenComponent";
 import FooterWithButtons from "./ui/FooterWithButtons";
 
-type Props = {
-  onClose: () => void;
+export type RemindEmailValidationInterface = {
   email: string;
+  validateEmail: () => void;
 };
 
 export default class RemindEmailValidationOverlay extends React.PureComponent<
-  Props
+  RemindEmailValidationInterface
 > {
-  private handleBackPress = () => {
-    this.props.onClose();
-    return true;
-  };
-
-  public componentDidMount() {
-    BackHandler.addEventListener("hardwareBackPress", this.handleBackPress);
-  }
-
-  public componentWillUnmount() {
-    BackHandler.removeEventListener("hardwareBackPress", this.handleBackPress);
-  }
-
   public render() {
     return (
-      <TopScreenComponent
-        customRightIcon={{
```

Comment:
As the invisio [here](https://projects.invisionapp.com/share/A9TQ5924R3W#/screens), the displayed component should have a cross icon which allows the user to close the displayed component and navigate:
- to previous screen if the condition is not yet satisfied
- to the new screen if the condition is satisfied